python/ray/scripts/scripts.py

```diff
@@ -335,19 +332,18 @@ def start(node_ip_address, redis_address, address, redis_port,
         logger.info(
             "\nStarted Ray on this node. You can add additional nodes to "
             "the cluster by calling\n\n"
-            "    ray start --redis-address {}{}{}\n\n"
+            "    ray start --address='{}'{}\n\n"
             "from the node you wish to add. You can connect a driver to the "
             "cluster from Python by running\n\n"
             "    import ray\n"
-            "    ray.init(redis_address=\"{}{}{}\")\n\n"
+            "    ray.init(address='auto'{})\n\n"
```

Comment:
Really when we do `address='auto'` it should be able to automatically detect the Redis password.